reactnative/package.json

```diff
@@ -17,7 +17,7 @@
   "dependencies": {
     "react": "16.8.3",
     "react-native": "0.60.5",
-    "magic-script-components-react-native": "git+https://github.com/magic-script/magic-script-components-react-native"
+    "magic-script-components-react-native": "../../magic-script-components-react-native"
```

Comment:
We should keep the remote dependency :-)

Reply:
good catch